src/components/Attachment/FileAttachment.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import prettybytes from 'pretty-bytes';
+// @ts-expect-error
```

Comment:
Dont think we need this anymore

Reply:
the ts expect error? still need because react-file-utils isn't typed yet